opm/input/eclipse/Schedule/GasLiftOpt.cpp

```diff
@@ -17,12 +17,40 @@
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#include <opm/input/eclipse/Schedule/GasLiftOpt.hpp>
+
+#include <cstddef>
+#include <map>
+#include <optional>
 #include <stdexcept>
+#include <string>
 
-#include <opm/input/eclipse/Schedule/GasLiftOpt.hpp>
+namespace {
+    std::optional<double> liftOptMaxRateIfDefined(const double x)
+    {
+        return (x > 0.0) ? std::optional { x } : std::nullopt;
+    }
+} // Anonymous namespace
 
 namespace Opm {
 
+bool GasLiftGroup::active(const RestartIO::RstGroup& rst_group)
+{
+    if (rst_group.glift_max_rate +
```

Comment:
could be return a+b != 0

Reply:
> could be return a+b != 0

Good point.  I've pushed an update to that effect.  It's not unlikely, though, that we'll revise this condition in the future.